pkg/apis/core/types.go

```diff
@@ -1600,7 +1600,7 @@ type LocalVolumeSource struct {
 	Path string
 }
 
-// Represents storage that is managed by an external CSI volume driver
+// Represents storage that is managed by an external CSI volume driver (Beta feature)
 type CSIPersistentVolumeSource struct {
```

Comment:
Was there a plan to add Attributes into CSIPersistentVolumeSource so we don't need to use annotations? I'd appreciate if the type was completed before marked as beta

Reply:
@jsafrane yes there is a separate PR for it, this can stall until it goes in. https://github.com/kubernetes/kubernetes/pull/58762